src/App.jsx

```diff
@@ -14,57 +15,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const toggleGood = good => {
```

Comment:
The method name 'toggleGood' should start with a verb to clearly indicate its action. Consider renaming it to something like 'handleToggleGood' or 'toggleSelectedGood' to better align with the checklist requirement: 'Method's name should start with a verb.'